3rdParty/CMakeLists.txt

```diff
@@ -106,3 +106,32 @@ add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/snappy/google-snappy-d53de18)
 ################################################################################
 
```

Comment:
This is ugly but it can not be helped:(

Copy-paste-leftovers like  ${NINJA_BUILD_DIR} could be removed.